superset/views/sql_lab.py

```diff
@@ -145,7 +146,7 @@ def post(self) -> FlaskResponse:  # pylint: disable=no-self-use
         )
         (
             db.session.query(TabState)
-            .filter_by(user_id=g.user.get_id())
+            .filter_by(user_id=get_user_id())
```

Comment:
Mixed types, though likely benign from the database's perspective—at least for MySQL.